salt/modules/virt.py

```diff
@@ -253,10 +255,7 @@ def _gen_xml(name,
         # TODO: make bus and model parameterized, this works for 64-bit Linux
         context['controller_model'] = 'lsilogic'
 
-    if kwargs.get('enable_vnc', True):
-        context['enable_vnc'] = True
-    else:
-        context['enable_vnc'] = False
+    context['enable_vnc'] = kwargs.get('enable_vnc', False)
```

Comment:
I know it's not yours, but I don't really like the `enable_vnc` option in the first place since that rules out the other graphic types supported by libvirt like spice. Refactoring it for a broader support would be good (I may get to it sooner or later)

Reply:
This changes the default value of `enable_vnc`. I'm not sure if you're ok with that.